src/sentry/static/sentry/app/components/events/errors.tsx

```diff
@@ -0,0 +1,243 @@
+          <li>
+            {t(
+              'If an event is reprocessed but did not change, we will not create the new version and not bill you for it (for now).'
+            )}
+          </li>
+          <li>
+            {t(
+              'If you have provided missing symbols please wait at least 1 hour before attempting to re-process. This is a limitation we will try to get rid of.'
+            )}
+          </li>
+        </ul>
+      </Body>
+      <Footer>
+        <ButtonBar gap={1}>
+          <Button onClick={this.onReprocessGroup}>
+            {t('Reprocess all events in issue')}
+          </Button>
+          <Button style={{marginLeft: space(1)}} onClick={this.onReprocessEvent}>
+            {t('Reprocess single event')}
+          </Button>
+          <Button style={{marginLeft: space(1)}} onClick={closeModal}>
+            {t('Cancel')}
+          </Button>
+        </ButtonBar>
+      </Footer>
+    </React.Fragment>
+  );
+
+  render() {
+    const {event, project} = this.props;
+    // XXX: uniqueErrors is not performant with large datasets
+    const errors =
+      event.errors.length > MAX_ERRORS ? event.errors : this.uniqueErrors(event.errors);
+    const numErrors = errors.length;
+    const isOpen = this.state.isOpen;
+    return (
+      <StyledBanner priority="danger">
+        <BannerSummary>
+          <StyledIconWarning />
+          <span>
+            {tn(
+              'There was %s error encountered while processing this event',
+              'There were %s errors encountered while processing this event',
+              numErrors
+            )}
+          </span>
+          <a data-test-id="event-error-toggle" onClick={this.toggle}>
+            {isOpen ? t('Hide') : t('Show')}
+          </a>
+        </BannerSummary>
+        <ErrorList
+          data-test-id="event-error-details"
+          style={{display: isOpen ? 'block' : 'none'}}
+        >
+          {errors.map((error, errorIdx) => (
+            <EventErrorItem key={errorIdx} error={error} />
+          ))}
+
+          {project.features.includes('reprocessing-v2') && (
+            <Button size="xsmall" onClick={this.onReprocessStart}>
+              {t('Try again')}
+            </Button>
+          )}
+        </ErrorList>
+      </StyledBanner>
+    );
+  }
+}
+
+const StyledBanner = styled(BannerContainer)`
+  margin-top: -1px;
+
+  a {
+    font-weight: bold;
+    color: ${p => p.theme.gray600};
+    &:hover {
+      color: ${p => p.theme.gray700};
+    }
+  }
+`;
+
+const StyledIconWarning = styled(IconWarning)`
+  color: ${p => p.theme.red400};
+`;
+
+// TODO(theme) don't use a custom pink
+const customPink = '#e7c0bc';
+
+const ErrorList = styled('ul')`
+  border-top: 1px solid ${customPink};
+  margin: 0 ${space(3)} 0 ${space(4)};
+  padding: ${space(1)} 0 ${space(0.5)} ${space(4)};
+
+  li {
+    margin-bottom: ${space(0.75)};
+    word-break: break-word;
+  }
+
+  pre {
+    background: #f9eded;
+    color: #381618;
```

Comment:
isn't there any color from the theme that you could use here?

Reply:
idk this is from the old code